pkg/task/restore.go

```diff
@@ -147,12 +147,24 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 		return errors.Annotate(berrors.ErrRestoreInvalidBackup, "contain tables but no databases")
 	}
 
+	restoreTS, err := client.GetTS(ctx)
+	if err != nil {
+		return err
+	}
+
+	sp := utils.BRServiceSafePoint{
+		BackupTS: restoreTS,
+		TTL:      utils.DefaultBRGCSafePointTTL,
```

Comment:
perhaps we should also make this configurable like backup?

Reply:
1. restore_raw doesn't have checksum logic. so we don't need to touch it.
2. I would like to make TTL a constant also in backup. because we know the default minimum gc life time is 10 minutes. we can set to 2 or 3 minutes, and the user seems never change these config?🤔️

Reply:
> I would like to make TTL a constant also in backup. 

Hmm perhaps file an issue or open a thread in SIG-Migrate so we can discuss it separately?

Reply:
Ok, I will open a thread in SIG-Migrate. How about merge it first so that won't block other PRs?